src/app/extensions/quoting/pages/quote-list/quote-list-page.component.ts

```diff
@@ -9,14 +9,27 @@ import { Quote, QuoteRequest } from '../../models/quoting/quoting.model';
   templateUrl: './quote-list-page.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class QuoteListPageComponent implements OnInit {
+export class QuoteListPageComponent implements OnInit, OnDestroy {
   loading$: Observable<boolean>;
-  quotes$: Observable<(Quote | QuoteRequest)[]>;
+  quotes: (Quote | QuoteRequest)[];
 
-  constructor(private quotingFacade: QuotingFacade) {}
+  private destroy$ = new Subject<void>();
+
+  constructor(private quotingFacade: QuotingFacade, private cd: ChangeDetectorRef) {}
 
   ngOnInit() {
-    this.quotes$ = this.quotingFacade.quotingEntities$();
+    this.quotingFacade
+      .quotingEntities$({ automaticRefresh: true })
+      .pipe(takeUntil(this.destroy$))
+      .subscribe(quotes => {
+        this.quotes = quotes;
+        this.cd.detectChanges();
```

Comment:
try to use ngrxPush pipeline instead ( https://v11.ngrx.io/guide/component/push )